scripts/index.js

```diff
@@ -1,47 +1,52 @@
-/**
- * Plays a song from the player.
- * Playing a song means changing the visual indication of the currently playing song.
- *
- * @param {String} songId - the ID of the song to play
- */
-function playSong(songId) {
-    // Your code here
+function createSongElement() {
+    const songsdiv = document.getElementById("songs");
+    for (let key of player.songs) {
+        getSongValuesFronId(key.id);
+        songsdiv.appendChild(document.createElement("div")).innerHTML = `<img class="songIMG" src="${coverArt}"><p class="songPara"><span class="songTitle">${title}</span> <span class="songArtist">By ${artist}</span> <span class="songAlbum">From ${album}</span> <span class="songDuration">${durationToMMSS(duration)}</span></p><img class="playButton" onclick="playTheSong(${key.id})" src="./images/playButton.png">`;
+    }
+    const songdivs = songsdiv.getElementsByTagName("div");
+    for (let key of songdivs) {
+        key.classList.add("eachSong");
+    }
 }
-
-/**
- * Creates a song DOM element based on a song object.
- */
-function createSongElement({ id, title, album, artist, duration, coverArt }) {
-    const children = []
-    const classes = []
-    const attrs = { onclick: `playSong(${id})` }
-    return createElement("div", children, classes, attrs)
+function createPlaylistElement() {
+    const playlistdiv = document.getElementById("playlists");
+    for (let key of player.playlists) {
+        getSongValuesFronId(key.songs[0]);
+        playlistdiv.appendChild(document.createElement("div")).innerHTML = `<img class="songIMG" src="${coverArt}"><p class="songPara"><span class="playlistName">${key.name}</span> <span class="playlistSongCount">${getNumOfSongsInPlaylist(key.id)} Songs</span> <span class="playlistDuration">Total duration ${durationToMMSS(playlistDuration(key.id))}</span></p><img class="playButton" onclick="playTheSong4Playlist(${key.id})" src="./images/playButton.png">`;
+    }
+    const playlistDiv = playlistdiv.getElementsByTagName("div");
+    for (let key of playlistDiv) {
+        key.classList.add("eachSong");
+    }
 }
-
-/**
- * Creates a playlist DOM element based on a playlist object.
- */
-function createPlaylistElement({ id, name, songs }) {
-    const children = []
-    const classes = []
-    const attrs = {}
-    return createElement("div", children, classes, attrs)
+function playTheSong(songID) {
+    getSongValuesFronId(songID);
+    document.getElementById("songDes").innerHTML = `Playing ${title} By ${artist} From The Album ${album}`;
```

Comment:
Bad.
innerHtml is for HTML (surprise...), try finding a different property for Text